app/Http/Requests/Settings/SetLoginRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests\Settings;
+
+use App\Exceptions\UnauthorizedException;
+use App\Http\Requests\Session\LoginRequest;
+use App\Legacy\AdminAuthentication;
+
+class SetLoginRequest extends LoginRequest
```

Comment:
As this class is new (and already legacy) I prefer to call it
```suggestion
class SetAdminLoginRequest extends LoginRequest
```
to clearly and undoubtedly express that this request is _only_ needed to manipulate the (legacy) admin account. I first assumed it might be used for the general case.

Of course, if one gives a second thought one can infer that because this request resides in `namespace App\Http\Requests\Settings`, but this is too inconspicuous for my taste.